gensim/models/nmf.py

```diff
@@ -649,8 +648,8 @@ def _solve_w(self):
 
         def error(WA):
             return (
-                0.5 * np.einsum('ij,ij', WA, self._W, optimize=True)
-                - np.einsum('ij,ij', self._W, self.B, optimize=True)
+                0.5 * np.einsum('ij,ij', WA, self._W)
+                - np.einsum('ij,ij', self._W, self.B)
```

Comment:
This expression could be on one line. But the function deserves a docstring: what is it calculating, what is `WA`?

Reply:
Will do